python/dgl/nn/pytorch/sparse_emb.py

```diff
@@ -170,9 +168,11 @@ def __call__(self, node_ids, device=th.device("cpu")):
             Target device to put the collected embeddings.
         """
         if not self._comm:
+            # embeddings are stored on the CPU
             emb = self._tensor[node_ids].to(device)
         else:
-            # will handle self._world_size = 0 or 1
+            # embeddings are stored on the GPU
+            # the following method will handle self._world_size = 0 or 1
```

Comment:
What's will happen if self._world_size != 0 or 1

Reply:
The `nccl.sparse_all_to_all_pull/push` methods are designed to perform communications across GPUs, but for world_size == 0 or 1, there is only one GPU so we don't have to do the **real** pull/push communication.

Reply:
I changed the comments to `the following method also covers self._world_size = 0 or 1`. Hope it would be clearer.

Reply:
Ah, I see. You should just mention the explanation in the comment for both pull and push method.
e.g.
```
The following method is designed to facilitate communication across multiple GPUs and can handle situations where only one GPU is present or no GPUs (a.k.a. self._world_size = 0 or 1) are available gracefully.
```